src/adagrams.js

```diff
@@ -1,7 +1,156 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    let letterPool = (
+      "A".repeat(9) +
+      "B".repeat(2) +
+      "C".repeat(2) +
+      "D".repeat(4) +
+      "E".repeat(12) +
+      "F".repeat(2) +
+      "G".repeat(3) +
+      "H".repeat(2) +
+      "I".repeat(9) +
+      "J" +
+      "K" +
+      "L".repeat(4) +
+      "M".repeat(2) +
+      "N".repeat(6) +
+      "O".repeat(8) +
+      "P".repeat(2) +
+      "Q" +
+      "R".repeat(6) +
+      "S".repeat(4) +
+      "T".repeat(6) +
+      "U".repeat(4) +
+      "V".repeat(2) +
+      "W".repeat(2) +
+      "X" +
+      "Y".repeat(2) +
+      "Z"
+    ).split("");
+    let hand = [];
+
+    for (let i = 0; i < 10; i++) {
```

Comment:
You're modifying your `letterPool` here. Make sure to make a deep copy first. In addition, consider how you could refactor such that you don't shuffle the pool 10 times.